src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-node/psbt-input-with-inscription.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box } from '@stacks/ui';
+import { truncateMiddle } from '@stacks/ui-utils';
+
+import { isUndefined } from '@shared/utils';
+
+import { openInNewTab } from '@app/common/utils/open-in-new-tab';
+import { OrdinalIcon } from '@app/components/icons/ordinal-icon';
+import { InscriptionPreview } from '@app/components/inscription-preview-card/components/inscription-preview';
+import { LoadingSpinner } from '@app/components/loading-spinner';
+import { useInscription } from '@app/query/bitcoin/ordinals/inscription.hooks';
+
+import { PsbtDecodedNodeLayout } from './psbt-decoded-node.layout';
+
+interface PsbtInputWithInscriptionProps {
+  address: string;
+  inputValue: string;
+  path: string;
+}
+export function PsbtInputWithInscription({
+  address,
+  inputValue,
+  path,
+}: PsbtInputWithInscriptionProps) {
+  const {
+    isLoading,
+    isError,
+    data: inscription,
+  } = useInscription(path.replace('/inscription/', ''));
+
+  if (isLoading)
+    return (
+      <Box my="loose">
+        <LoadingSpinner />
+      </Box>
+    );
+  if (isError || isUndefined(inscription))
+    return (
+      <PsbtDecodedNodeLayout
+        hoverLabel={address}
+        image={<OrdinalIcon />}
+        subtitle={truncateMiddle(address)}
+        subValue="# Unknown"
+        title="No data"
+        value={`- ${inputValue}`}
+      />
+    );
+
+  return (
+    <PsbtDecodedNodeLayout
+      hoverLabel={address}
+      image={<InscriptionPreview inscription={inscription} height="40px" width="40px" />}
+      subtitle={truncateMiddle(address)}
+      subValue={`#${inscription.number}`}
+      subValueAction={() => openInNewTab(inscription.infoUrl)}
+      title="Ordinal inscription"
+      value={`- ${inputValue}`}
```

Comment:
https://symbl.cc/en/2212 🙃 

```suggestion
      value={`−${inputValue}`}
```

Also started reading into formatting of positive negative numbers. Rule of thumb is that when there's a mathematical operation taking place, they're spaced with a ` `, but not when describing a negative unit.